impl/src/main/java/org/ehcache/config/builders/ExpiryPolicyBuilder.java

```diff
@@ -75,9 +81,9 @@ public static ExpiryPolicyBuilder expiry() {
     return new ExpiryPolicyBuilder();
   }
 
-  private Duration create = ExpiryPolicy.INFINITE;
-  private Duration access = null;
-  private Duration update = null;
+  private BiFunction<Object, Object, Duration> create = (key, value) -> ExpiryPolicy.INFINITE;
```

Comment:
This type should really have proper generic typing if we're going to expand in to key/value sensitive expiry policies.

Reply:
What do you mean?

Reply:
See: https://github.com/henri-tremblay/ehcache3/pull/2